Addressed issue #1599 : Updated tool page on OSCAL pages (#1657)
usnistgov/OSCAL: docs/content/tools/_index.md

- converting *into* JSON - [XML to JSON](https://github.com/usnistgov/OSCAL/tree/main/json)
- converting *into* XML - [JSON to XML](https://github.com/usnistgov/OSCAL/tree/main/xml)
 
## NIST's OSCAL Application Frameworks, Tools and Libraries

> Valid OSCAL is open-ended in application. Some of the tools described provide validation and conversion, while others do not, presuming that inputs are already valid.

Reviewer: So we are not including `oscal-cli` yet? If so, we should add it.

Author: Added

> | Name | Provider/Developer | Description | Type |

Reviewer: Also, I forgot, I think it is best we get rid of the `Provider/Developer` column and adjust the remaining rows since there will only be one. No need to repeat it anymore. :-)
```suggestion
| Name | Description | Type |
```

Author: Removed

|:---|:---|:---|:---|

> | [OSCAL Deep Diff](https://github.com/usnistgov/oscal-deep-diff) | NIST OSCAL Project | A Typescript-based CLI application and library that produces machine readable and human-consumable comparisons of JSON OSCAL artifacts. | open source |

Reviewer: Removing provider column here too.
```suggestion
| [OSCAL Deep Diff](https://github.com/usnistgov/oscal-deep-diff) | NIST OSCAL Project | A Typescript-based CLI application and library that produces machine readable and human-consumable comparisons of JSON OSCAL artifacts. | open source |
```
```suggestion
| [OSCAL Deep Diff](https://github.com/usnistgov/oscal-deep-diff) | A Typescript-based CLI application and library that produces machine readable and human-consumable comparisons of JSON OSCAL artifacts. | open source |
```

Author: Removed

> | [OSCAL Java Library](https://github.com/usnistgov/liboscal-java) | NIST OSCAL Project | A Java-based programming API for reading and writing content conformant to the OSCAL XML, JSON, and YAML based models. | open source |

Reviewer: Removing provider column here too and add `oscal-cli` entry.

```suggestion
| [oscali-cli](https://github.com/usnistgov/oscal-cli/) | A command line interface, that performs common operations on [Open Security Controls Assessment Language](https://pages.nist.gov/OSCAL/) (OSCAL) and [Metaschema](https://github.com/usnistgov/metaschema) content. | open source |
| [OSCAL Java Library](https://github.com/usnistgov/liboscal-java) | A Java-based programming API for reading and writing content conformant to the OSCAL XML, JSON, and YAML based models. | open source |
```

Author: Done.

> | [XSLT Tooling](https://github.com/usnistgov/oscal-tools/tree/master/xslt) | NIST OSCAL Project | A variety of Extensible Stylesheet Language (XSL) Transformations (XSLT), Cascading Style Sheets (CSS), and related utilities for authoring, converting, and publishing OSCAL content in various forms. | open source |

Reviewer: Can we update the branch please?

```suggestion
| [XSLT Tooling](https://github.com/usnistgov/oscal-tools/tree/main/xslt) | NIST OSCAL Project | A variety of Extensible Stylesheet Language (XSL) Transformations (XSLT), Cascading Style Sheets (CSS), and related utilities for authoring, converting, and publishing OSCAL content in various forms. | open source |
```
